_data/retail.yml

```
@@ -188,6 +188,16 @@ websites:
     twitter: flipkartsupport
     facebook: flipkart
 
+  - name: Galaxus
+    url: https://www.galaxus.ch/
+    img: galaxus.png
+    tfa:
+      - totp
+    doc: https://www.galaxus.ch/en/page/6589
+    regions:
+      - ch
+      - de
```

```suggestion
      - de
      - li
```
Galaxus, like Digitec, also [serves Liechtenstein](https://www.galaxus.ch/en/wiki/556).